OpenRA.Mods.Cnc/Traits/PaletteEffects/LightPaletteRotator.cs

```diff
@@ -18,8 +18,24 @@ namespace OpenRA.Mods.Cnc.Traits
 	[Desc("Palette effect used for blinking \"animations\" on actors.")]
 	class LightPaletteRotatorInfo : ITraitInfo
 	{
+		[Desc("Palettes this effect should not apply to.")]
 		public readonly HashSet<string> ExcludePalettes = new HashSet<string>();
 
+		[Desc("'Speed' at which the effect cycles through palette indices.")]
+		public readonly float TimeStep = .5f;
+
+		[Desc("Palette index to map to rotating colors.")]
+		public readonly int ModifyIndex = 103;
+
+		[Desc("First palette index of rotating color range.")]
+		public readonly int RotationStartIndex = 230;
```

Comment:
Would there be any value in specifying this as a list of indices to cycle through instead a contiguous range?